requirements.txt

```diff
@@ -0,0 +1,4 @@
+mritopng==2.1
```

Comment:
Do you really need to have the project itself as a dependency? Typically you'd want to sync this repo, make your changes and use your version, in which case using `python setup.py install` will do this for you locally.

Reply:
Just forgot to remove it from the pip freeze output, I'll remove it